README.md

```diff
@@ -76,4 +79,5 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification.
+Contributions of any kind welcome!
```

Comment:
Revert

Reply:
This was done by prettier, not by me

Reply:
No it wasn't because our prettier config isn't set up like that. Regardless, it doesn't take away that you should revert it.

Reply:
I know